hw/top_earlgrey/syn/chip_earlgrey_asic.sdc

```diff
@@ -182,82 +182,173 @@ if {!$IS_CDC_RUN} {
 # SPI System Parameters             #
 #####################################
 
-# routing delay from external component to device
-set PCB_DEL 1
+# Routing delay from external component to device
+# Represents approximately 5 inches of trace
+set PCB_DEL 0.85
 
-# external spi host setup
-set HOST_SETUP_DEL 4
+# Max skew between signals. Represents approximately 3 inches.
+set PCB_SKEW 0.51
+
```

Comment:
@toshaq Does this look reasonable? Do these need an adjustment?

Reply:
Yeah this looks good. 3 inches on both side of interposer is a reasonable assumption for now